lib/index.js

```diff
@@ -65,6 +65,7 @@ Elevio.prototype.identify = function(identify) {
   if (email) user.email = email;
   if (name) user.name = name;
   if (plan) user.plan = [plan];
+  if (plan) user.groups = [plan];
```

Comment:
@duellsy thanks for the PR -- qq here. What is the impact of this feature? It does seem like this should be only augmentative data so it wouldn't have any "breaking" changes but just want to make sure that users of elevio would all welcome this new change and that it shouldn't be gated behind an opt-in setting.